packages/dev/core/src/Loading/Plugins/babylonFileLoader.ts

```diff
@@ -740,7 +740,13 @@ SceneLoader.RegisterPlugin({
                         }
 
                         // Skeleton ?
-                        if (parsedMesh.skeletonId > -1 && parsedData.skeletons !== undefined && parsedData.skeletons !== null) {
+                        if (

```

Comment:
I think you need the same on the morphtargetmanager just after this block ???

Reply:
The morphtargetmanager serializes uniqueIds, so it's always a number: https://github.com/BabylonJS/Babylon.js/blob/master/packages/dev/core/src/Morph/morphTargetManager.ts#L282